src/libsync/progressdispatcher.cpp

```diff
@@ -189,9 +189,17 @@ void ProgressInfo::adjustTotalsForFile(const SyncFileItem &item)
         return;
     }
 
-    _fileProgress._total += item._affectedItems;
-    if (isSizeDependent(item)) {
-        _sizeProgress._total += item._size;
+    // Update _currentItems map
+    setProgressItem(item, 0);
```

Comment:
Mmmh, that means that during treewalk we allocate entries in `_currentItems` for all items now. Would prefer if that wasn't necessary. Why is this necessary?

Reply:
I've removed the changes to `ProgressInfo::adjustTotalsForFile`.